.github/workflows/actionlint.yml

```diff
@@ -0,0 +1,81 @@
+# This file is synced from the `.github` repository, do not modify it directly.
+name: Workflow Syntax
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/*.ya?ml'
+  pull_request:
+    paths:
+      - '.github/workflows/*.ya?ml'
+
+defaults:
+  run:
+    shell: bash -xeuo pipefail {0}
+
+concurrency:
+  group: "actionlint-${{ github.ref }}"
+  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
+
+env:
+  HOMEBREW_DEVELOPER: 1
+  HOMEBREW_NO_AUTO_UPDATE: 1
+  HOMEBREW_NO_ENV_HINTS: 1
+
+permissions: {}
+
+jobs:
+  workflow_syntax:
+    if: github.repository_owner == 'Homebrew'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Homebrew
+        id: setup-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+        with:
+          core: false
+          cask: false
+          test-bot: false
+
+      - name: Install tools
+        run: brew install actionlint shellcheck zizmor
```

Comment:
Will `actionlint` call overlap with `brew style` or `brew tap-syntax` in any of those repos?

Reply:
Probably for `brew style`, not for `brew tap-syntax`. But running it inside `brew style` doesn't seem to work properly anyway -- we're getting new `actionlint` errors now. (Unless we weren't running `brew style` or `brew tap-syntax` where that happened, but I don't think that was the case.)

Reply:
Actually, I just double checked:

- it won't overlap, because we skip `actionlint` for repos with an `actionlint.yml` workflow
- the new errors are from the `-ignore` we pass in `brew style`, so the new errors are not evidence of `brew style` not working properly.